test/regress_ws.c

```diff
@@ -177,7 +177,8 @@ receive_ws_msg(struct evbuffer *buf, size_t *out_len, unsigned *options)
 		return NULL;
 
 	mask_key = data + header_len - 4;
-	for (size_t i = 0; mask && i < payload_len; i++)
+	size_t i;
```

Comment:
Now it complains:

```
/home/runner/work/libevent/libevent/dist/libevent-2.2.0-alpha-dev/ws.c: In function ‘get_ws_frame’:
/home/runner/work/libevent/libevent/dist/libevent-2.2.0-alpha-dev/ws.c:244:3: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
   int i;
```

What was the error on centos 7 initially?

Reply:
Hi :wave: 

```
  CC       ws.lo
ws.c: In function 'get_ws_frame':
ws.c:244:3: error: 'for' loop initial declarations are only allowed in C99 mode
   for (int i = 0; i < payload_len; i++) {
   ^
ws.c:244:3: note: use option -std=c99 or -std=gnu99 to compile your code
```

The command used for building:
```
./autogen.sh ; ./configure --prefix=/usr --libdir=/usr/lib64 ; make all
```

```
$ gcc --version
gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
```

Reply:
Ok, let's move this variable to the beginning of the funciton.

Reply:
did that - waiting for automated tests